### DISCUSSION TITLE
Skydimo monitor backlights from Aliexpress

### DISCUSSION THREAD

**Discussion by Hummtaro** [hyperion-project/hyperion.ng#1693] [Show and tell]
Do you people know those Skydimo monitor backlights? I'm just here to tell that those work fine with Hyperion.
I recently got one for my 34" ultrawide monitor and i really like that it connects to a single USB port. No big controller box, no seperate power supply, no wiring hassle.

It only took me some time to figure out the right config.. Controller type is adalight but the hardware LED count needs to be 255 for some reason, even if way less LEDs are present. The 34" version has 71 LEDs total, 41 horizontal and 15 vertical.

Whoever searches the net for compatibility of those Skydimo lights may find this post. 👋

**Comment by novikov9999** [comment]
Hi. That's very nice to hear. Can you share the configuration?

**Reply by Hummtaro** [reply]
LED Controller:
Controller type: adalight
Streaming Protocol: Standard
Hardware LED count: 255

LED Layout:
Top: 41
Left: 15
Right: 15
Input position: 56
Edge Gap: 25%

Color channel adjustments:
White: rgb(255,255,255)
Red: rgb(205,4,3)
Green: rgb(90,255,0)
Blue: rgb(5,10,255)
Cyan: rgb(80,255,205)
Magenta: rgb(255,0,255)
Yellow: rgb(255,255,0)

Thats for the 34" Ultrawide version, for the smaller ones the layout would slightly differ.
The color adjustment is not quite on point but good enough for me. :)

**Reply by Hummtaro** [reply]
Oh, i just updated to the latest nightly and now there is a Skydimo Mode! So for the next release the settings (for 34") are:

LED Controller:
Controller type: adalight
Streaming Protocol: Skydimo Mode
Hardware LED count: 71

**Comment by twinstef** [comment]
hello,
where can we find  the latest nightly on github - i cant find any way.

**Reply by Hummtaro** [reply]
As far i know there is no official nightly download. Here is an up to date version built by github: https://github.com/Hummtaro/hyperion.ng/releases/tag/beta

Controller type: skydimo

**Reply by Danyuiel** [reply]
Please update build for windows

**Reply by Hummtaro** [reply]
The build i linked is just two weeks old, there are no changes since then. What do you want?

**Comment by styloren** [comment]
Forgive me if I'm not asking this correctly, first time actually on Git for more than just downloading some files. 

So I stumbled across this b/c I just got the Skydimo lights and was wary about running the native software. I got the initial configuration set up running as an adalight controller, however my setup is two 34" ultrawide monitors stacked and it seems like I can only run one monitor at a times? Hyperion sees both COM units and I can address each of them, but how/can I run both strips like one giant frame?

**Reply by Hummtaro** [reply]
I don't know how to capture two local screens..
You'd better create a new discussion for your question..

**Reply by styloren** [reply]
Cool, thanks!

**Comment by styloren** [comment]
Hey thanks for getting back to me. I was able to get them setup with the generic adalight controller and then had to play around with the LED layout.

Sent from [Proton Mail](https://proton.me/mail/home) for iOS.

-------- Original Message --------
